cypress/integration/pages/frontPage/tests.js

```diff
@@ -83,6 +85,27 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
               .should('be.visible');
           });
         });
+
+        if (serviceHasIndexAlsos(service)) {
+          it('should contain Index Alsos at a mobile view', () => {
+            cy.viewport('iphone-5');
+            cy.get('section li')
+              .eq(0)
+              .within(() => {
+                cy.get('div div div a')
+                  .eq(0)
+                  .within(() => {
+                    cy.get('span').then($el => {
+                      if ($el.length > 1) {
+                        cy.get('svg').should('be.visible');
+                      } else {
+                        expect($el).not.to.have.descendants('svg');
```

Comment:
why are we looking for a `svg` if there are no index alsos?

Reply:
We're not. This logic is written in a way that it expects to find `index-also` on the page. The current logic within `index-also` means that there will not be an SVG present if the `span` length is `1`.